ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries/new.js

```diff
@@ -13,12 +16,22 @@ export default class ScopesScopeCredentialStoresCredentialStoreCredentialLibrari
    * @return {CredentialLibraryModel}
    */
   model() {
-    const { id: credential_store_id, type } = this.modelFor(
+    const { id: credential_store_id } = this.modelFor(
       'scopes.scope.credential-stores.credential-store'
     );
+
     return this.store.createRecord('credential-library', {
-      type,
+      type: TYPE_CREDENTIAL_LIBRARY_VAULT_GENERIC,
```

Comment:
We still need to get the user-selected type from the query param, don't we?  Generic is the default if type is not provided.

Reply:
I was going to add in that functionality in the next PR but I can add it here